Parsers/ASimAuditEvent/Tests/SentinelOne_AsimAuditEvent_SchemaTest.csv

```diff
@@ -0,0 +1,74 @@
+﻿Result
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [ObjectId]"
+"(2) Info: Missing optional alias [Application] aliasing non-existent column [TargetAppName]"
+"(2) Info: Missing optional alias [Process] aliasing non-existent column [ActingProcessName]"
+"(2) Info: Missing optional field [ActingAppId]"
+"(2) Info: Missing optional field [ActingAppName]"
+"(2) Info: Missing optional field [ActingAppType]"
+"(2) Info: Missing optional field [ActorOriginalUserType]"
+"(2) Info: Missing optional field [ActorScopeId]"
+"(2) Info: Missing optional field [ActorScope]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [ActorUserAadId]"
+"(2) Info: Missing optional field [ActorUserSid]"
```

Comment:
is any of the above Actor information is available?

Reply:
No